examples/image_list/lib/main.dart

```diff
@@ -0,0 +1,148 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+import 'dart:typed_data';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart' show rootBundle;
+import 'package:flutter/services.dart';
+
+/// An example that sets up local http server for serving single
+/// image, creates single flutter widget with five copies of requested
+/// image and prints how long the loading took.
+///
+/// This is used in [$FH/flutter/devicelab/bin/tasks/image_list_reported_duration.dart] test.
+Future<void> main() async {
+  final HttpServer httpServer =
+      await HttpServer.bind(InternetAddress.anyIPv6, 0);
+  final int port = httpServer.port;
+  print('Listening on port $port.');
+
+  httpServer.listen((HttpRequest request) async {
+    final ByteData byteData = await rootBundle.load('images/coast.jpg');
```

Comment:
Is this the right place for this?  It seems like the test will be noisy if we try to do this much on each request.

Reply:
There should be no requests to this server other than from the test itself asking for the images, no?

Reply:
Right, but on each request you're going to see differences because you're either loading the asset or getting a cached version of it - I believe `.load` doesn't do caching though.

Reply:
Ah, in the test I always use different uris (localhost/1, localhost/2, ...) to avoid caching problem.

Reply:
Sorry, misunderstood your point. Addressed it now in 0xf6f9d80c.